react_ws_src/src/views/ttt/GameMain.js

```diff
@@ -339,5 +324,9 @@ export default class SetName extends Component {
 	}
 
 
-
+	animate_cell(cell_id) {
```

Comment:
This could use some formatting, but I didn't want to reformat the whole file. IMO the project should have `prettier` that runs on commit and in CI as a check. It would help clean up the formatting changesets and keep everything consistent.